recipes/limix-plot/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "limix-plot" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9a73d36eed7cc3df893a5923d404410f06b08adf6438a1398b69d25bff86df8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python > 3
```

I'm not sure if the space after the `>` works everywhere in `conda-build`:
```suggestion
    - python >3
```